src/index.js

```diff
@@ -15,12 +15,107 @@ const unexpectedEnzyme = {
       identify: function(reactWrapper) {
         return reactWrapper && reactWrapper instanceof ReactWrapper;
       },
-      inspect: function(reactWrapper, depth, output) {
-        const inspected = reactWrapper.exists()
-          ? reactWrapper.getElement()
-          : reactWrapper.root().html();
+      inspect: function(reactWrapper, depth, output, inspect) {
+        if (!reactWrapper.exists()) {
+          return output.jsKeyword('null');
+        }
+
+        if (reactWrapper.length > 1) {
+          return output.appendItems(reactWrapper.map(node => node), '\n');
```

Comment:
Not sure if there is any benefit on using [#getNodes](https://github.com/airbnb/enzyme/blob/master/docs/api/ReactWrapper/getNodes.md)

Reply:
I think the above will actually map to new React Wrappers which I need.